package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "mvsutils",
-  "version": "1.0.9",
+  "version": "1.1.0",
```

It doesn't appear that we have a semver minor change in this PR, should just the patch version be increased?

```suggestion
  "version": "1.0.10",
```